docs/content/users/debugging-profiling/step-debugging.md

```diff
@@ -126,6 +126,7 @@ Here are basic steps to take to sort out any difficulty:
 * Now click the “listen” button on PhpStorm to start listening for connections.
 * `ddev ssh` and try the `telnet host.docker.internal 9003` again. It should connect. If not, maybe PhpStorm is not listening, or not configured to listen on port 9003?
 * Check to make sure that Xdebug is enabled. You can use `php -i | grep -i xdebug` inside the container, or use any other technique you want that gives the output of `phpinfo()`, including Drupal’s `admin/reports/status/php`. You should see `with Xdebug v3` and `php -i | grep xdebug.mode` should give you `xdebug.mode => debug,develop => debug,develop"`.
+* If you're running WSL2 with Docker CE and VS Code on Windows check if `xdebug.discover_client_host` is on. If it is off add `xdebug.discover_client_host=1` to `.ddev/php/my-php.ini` in your project directory.
```

Comment:
This hasn't ever been necessary before... I think you may be running into a different problem.